stan/math/rev/core/operator_division.hpp

```diff
@@ -96,11 +100,15 @@ inline var operator/(const var& a, const var& b) {
  *
  * \f$\frac{\partial}{\partial x} (x/c) = 1/c\f$.
  *
+ * @tparam Var value type of a var
+ * @tparam Arith An arithmetic type
  * @param a Variable operand.
  * @param b Scalar operand.
  * @return Variable result of dividing the variable by the scalar.
  */
-inline var operator/(const var& a, double b) {
+template <typename Var, typename Arith, require_var_t<Var>...,
+          require_arithmetic_t<Arith>...>
+inline var operator/(Var&& a, Arith b) {
```

Comment:
[optional]
The right doc for `a` is `dividend`, for `b` is `divisor`, and for the result is `quotient`.

Reply:
Word!